ci/kmm-kmod-dockerfile.yaml

```diff
@@ -17,7 +17,8 @@ data:
 
     WORKDIR /usr/src
 
-    RUN git clone https://github.com/kubernetes-sigs/kernel-module-management.git
+    RUN grep super-secret-value /run/secrets/build-secret/ci-build-secret && \
+        git clone https://github.com/kubernetes-sigs/kernel-module-management.git
```

Comment:
```suggestion
    RUN grep super-secret-value /run/secrets/build-secret/ci-build-secret
    RUN git clone https://github.com/kubernetes-sigs/kernel-module-management.git
```

Reply:
I don't mind by why? I would like to think about it as `git clone --auth ...` so it makes sense to me to put in in the same layer.

Reply:
Fixed.